integration/scenario.go

```diff
@@ -210,3 +211,20 @@ func (s *Scenario) RunTailscaleUp(
 
 	return fmt.Errorf("failed to up tailscale node: %w", errNoNamespaceAvailable)
 }
+
+func (s *Scenario) GetIPs(namespace string) ([]netip.Addr, error) {
```

Comment:
Should we have GetIPv4 and GetIPv6?

Reply:
Can add helpers to filter after, but since netip holds both in the same type, we might as well get all in one command call since that's expensive